src/pages/Trade/Trade.tsx

```diff
@@ -86,12 +74,7 @@ export const Trade = () => {
           overflowY='auto'
         >
           <Card variant='unstyled' width='full'>
-            <Card.Header position='sticky' top={0} bg={headerBg} zIndex={2}>
-              <Card.Heading>
-                <Text translation='dashboard.recentTransactions.recentTransactions' />
-              </Card.Heading>
-            </Card.Header>
-            <TransactionHistoryList txIds={txIds} useCompactMode={true} />
+            <RecentTransactions />
```

Comment:
use the same component as the dashboard for the trade page - FYI @reallybeard this looks visually more or less the same, but only shows the most recent 10 txs, as per the dashboard.